include/grpcpp/ext/orca_service.h

```diff
@@ -48,38 +49,16 @@ class OrcaService : public Service {
     }
   };
 
-  explicit OrcaService(Options options);
-
-  // Sets or removes the CPU utilization value to be reported to clients.
-  void SetCpuUtilization(double cpu_utilization);
-  void DeleteCpuUtilization();
-
-  // Sets of removes the memory utilization value to be reported to clients.
-  void SetMemoryUtilization(double memory_utilization);
-  void DeleteMemoryUtilization();
-
-  // Sets of removes the QPS value to be reported to clients.
-  void SetQps(double qps);
-  void DeleteQps();
-
-  // Sets or removed named utilization values to be reported to clients.
-  void SetNamedUtilization(std::string name, double utilization);
```

Comment:
I think we need to add these methods on `ServerMetricRecorder`, since they already exist here.  If we remove them here and don't add them there, then we're removing functionality that already exists in this API.